qa/workunits/rbd/rbd_mirror_helpers.sh

```diff
@@ -671,6 +671,41 @@ wait_for_image_replay_stopped()
     wait_for_image_replay_state "${cluster}" "${pool}" "${image}" stopped
 }
 
+test_image_replayer_status()
+{
+    local cluster=$1
+    local pool=$2
+    local image=$3
+    local test_state=$4
+    local status_result
+
+    status_result=$(rbd --cluster "${cluster}" mirror image status ${pool}/${image} | grep \"replay_state\":\"${test_state} || return 1)
```

Comment:
we already have `test_status_in_pool_dir` and `wait_for_status_in_pool_dir` that check `rbd mirror image status` output. Couldn't they be used instead of adding the new ones?

Reply:
After some looking, I see what you mean. I didn't notice specifying a value for 5th arg `local description_pattern="$5"` would take care of that. Nice catch -- thanks!